tests/de/light_HassLightSet.yaml

```diff
@@ -138,6 +138,9 @@ tests:
       - Ändere die Helligkeit der ganzen Lichter im Schlafzimmer auf 11
       - Ändere die Helligkeit der kompletten Lichter im Schlafzimmer auf 11
       - Ändere die Helligkeit der sämtlichen Lichter im Schlafzimmer auf 11
+      - Dimme das Licht im Esszimmer auf 100%
+      - Setze die Helligkeits des Lichts im Esszimmer auf 100%
+      - Setze die Helligkeit des Lichtes im Esszimmer auf 100%
```

Comment:
Please also add test sentences for the newly introduced sentences for changing the color you defined earlier.